cmd/etcd-launcher/main.go

```diff
@@ -37,45 +39,44 @@ type envConfig struct {
 }
 
 type etcdCluster struct {
-	config      *envConfig
+	config      *config
 	client      *clientv3.Client
 	localClient *clientv3.Client
 }
 
 func main() {
 	// normal workflow, we don't do migrations anymore
 	e := &etcdCluster{}
-	err := e.getConfigFromEnv()
+	err := e.parseConfigFlags()
 	if err != nil {
 		log.Fatalf("failed to get launcher configuration: %v", err)
 	}
 
+	logOpts := kubermaticlog.NewDefaultOptions()
+	rawLog := kubermaticlog.New(logOpts.Debug, logOpts.Format)
+	log := rawLog.Sugar()
```

Comment:
This variable collides with the `"log"` imported package

Reply:
The imported package is aliased, so it should be ok?